resources/wikia/modules/cookiePolicy.js

```diff
@@ -0,0 +1,55 @@
+/* global GlobalNotification */
+require(['wikia.cookies', 'wikia.geo'], function (cookies, geo) {
+	'use strict';
+
+	/**
+	 * Initialize JS for whether or not to show a banner notifying European Union users that we use cookies
+	 * on our site, per EU law. Users can dismiss the notification and will not be shown again, unless
+	 * they clear their cookies.
+	 */
+	function initCookieNotification() {
+		if (shouldShowBanner()) {
+			showBanner();
+		}
+	}
+
+	/**
+	 * Check if user is from the EU and hasn't seen the banner yet
+	 * @returns {boolean}
+	 */
+	function shouldShowBanner() {
+		var fromEU = geo.getContinentCode() === 'EU';
+
+		return (fromEU && !hasSeenBanner());
+	}
+
+	/**
+	 * Display the cookie policy message to the user
+	 */
+	function showBanner() {
+		var message = $.msg('oasis-eu-cookie-policy-notification-message') +
```

Comment:
Taking into account that there is no way to create link with ```target="_blank"``` from wikitext it'd be hard to escape translation string. We cannot use simply use ```some text here <a href="http://example.com">some text here</a>``` because after escaping it won't work.

Possible solutions:
* add ```target="_blank"``` via js - hacky solution
* remove ```target="_blank"``` and load message through AJAX - this one needs to be accepted by PO
* extract URL into separate message key, concatenate strings in the same way as it was done originally, escaped all translated messages. We will end up with:
````
var message = mw.message('oasis-eu-cookie-policy-notification-message').escaped() +
'<a href="mw.message('oasis-eu-cookie-policy-notification-link-url').escaped()" target="_blank">' +
mw.message('oasis-eu-cookie-policy-notification-link-text').escaped() +
'</a>';
````
This one can be a small problem for i18n but would be more clear from technical point of view, that's why I prefer this one.

Please ping me if you find different solution.